### DISCUSSION TITLE
What is difficult to understand?

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#32] [General]
Is there anything in the documentation or design that is difficult to understand? Please feel free to chat.

**Comment by kevinram** [comment]
Hi.  I really like winvind.   Its far better than any of the AHK solutions I've tried.  My biggest problem is reassigning the short cuts for the modes.  For example, instead of ESC-LEFT, ESC-DOWN, ESC-RIGHT, as the shortcuts to start modes, I would like to use F15, F16, and F17.  Could you past a vindrc that does this?   Thanks!  Kevin

**Reply by pit-ray** [reply]
Thanks for using it.
Please refer to #91, #42.
If the answers at Issues are hard to understand, please let me know.
